src/cli.py

```diff
@@ -0,0 +1,46 @@
+import sys
+import click
+from pathlib import Path
+from src.comparsion import compare_and_output
+
+def validate_paths(input_path: str, output_path: str) -> None:
+    input_path = Path(input_path)
+    
+    if not input_path.is_dir():
+        raise FileNotFoundError(f"Input path does not exist: {input_path}")
+    
+    if output_path:
+        output_path = Path(output_path)
+        if not output_path.is_dir():        
+            output_path.mkdir(parents = True, exist_ok = True)
+            print(f"Attention: new folder created at: {output_path}")
+
+    if not output_path:
+        output_path = Path(Path.cwd(), "output")
+
+        if not output_path.is_dir():        
+            output_path.mkdir(parents = True, exist_ok = True)
+            print("Default output folder created at:", output_path)
+
+    return input_path, output_path
+
+@click.command()
+@click.option('--input', type = str)
```

Comment:
It might be better to have a 'help' parameter for each option to provide a brief explanation. For example:
`@click.option('--input', type = str, help = ‘Directory containing the files to be compared')`